2-mandatory/4-stocks.js

```diff
@@ -49,6 +62,18 @@ function getAveragePrices(closingPricesForAllStocks) {
 */
 function getPriceChanges(closingPricesForAllStocks) {
     // TODO
+     let priceChangeArray = [];
+     for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+       priceChangeArray.push(
+         Number(
+           (
+             -closingPricesForAllStocks[i][0] +
+             closingPricesForAllStocks[i][closingPricesForAllStocks.length - 1]
+           ).toFixed(2)
```

Comment:
Nice solution, well done. Your ordering on this line is a little confusing, this would make it more readable (last stock - first stock) :
`closingPricesForAllStocks[i][closingPricesForAllStocks.length - 1]  - closingPricesForAllStocks[i][0]`